flow/src/test/java/com/sequenceiq/flow/controller/FlowControllerTest.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.flow.controller;
```

Comment:
You can use MockMvc also to test controller layer which will help to test the endpoint with expected Http request.

Reply:
Thank you, Adarsh. But I don't think Flow is a springboot project and it is not possible to do a MockMVC with the web context imported from a different project.